### PR TITLE
Added Domain Model And Saved Developer Images Locally

### PR DESCRIPTION
**Pull request by ndalliard**
Please @Tschanzt review what I've done:

- Instead of saving the images of the [developers images here](https://s3.eu-central-1.amazonaws.com/travisupload/srs/team-list.html) on imgur.com I decided to save them locally on our server.
- Added the [Domain Model](https://s3.eu-central-1.amazonaws.com/travisupload/srs/domain_model.html) to the maven project site.

@imscaradh When this pull request will be merged will travis automatically rebuild the [maven site](https://s3.eu-central-1.amazonaws.com/travisupload/srs/index.html)?

@dansel91 Should I remove the title "SRS Store" on the Domain Model - it would look a little bit cleaner?!

### CODE REVIEW

**Comment by imscaradh** [comment]
@ndalliard the artifact upload to the S3 bucket will only performed on the master branch. Therefore the Travis build of your doc branch won't upload anything, but after merging back this branch to the master you should see your changes.

**Comment by ndalliard** [comment]
Thanks for clarifying.

File: src/site/apt/domain_model.apt [hunk 1]
```diff
@@ -1,25 +1,24 @@
-		----------
-		Domain Model
-		----------
-		due1
-		----------
-		October 03, 2015
```
Comment (imscaradh) [inline src/site/apt/domain_model.apt]: @ndalliard I would propose to leave those indents as they are

Comment (ndalliard) [inline src/site/apt/domain_model.apt]: @imscaradh I already did that in this commit: https://github.com/imscaradh/srs/commit/0cb26af9e12451d2587b150059c5836328fc8086 . The only difference is that I replaced the tabs with spaces. I read somewhere that spaces are better than tabs.

**Comment by ndalliard** [comment]
I just saw in the last commits https://github.com/imscaradh/srs/commit/f97448740772d04e5ba45b941fa892719fca9ad2 from @due1, that he changed the modification date of the files.

I should do that for the domain model and the developer list also right? Do I **add** my name and the date below him or **delete** his name and add my name with the corresponding date?

**Comment by imscaradh** [comment]
That's why I thought it would be better not to modify this section
(conflicts). We don't need modification informations inside a file. We are
using git for that!

I just saw in the last commits f974487
<https://github.com/imscaradh/srs/commit/f97448740772d04e5ba45b941fa892719fca9ad2>
from @due1 <https://github.com/due1>, that he changed the modification date
of the files.

I should do that for the domain model and the developer list also right? Do
I *add* my name and the date below him or *delete* his name and add my name
with the corresponding date?

—
Reply to this email directly or view it on GitHub
<https://github.com/imscaradh/srs/pull/29#issuecomment-147296271>.

**Comment by Tschanzt** [comment]
@ndalliard @imscaradh wouldn't the point of these files be that they would be used by people that might not have git access? Because then modifying these sections would make sense.

**Comment by imscaradh** [comment]
Ah, is this something that would be displayed inside the maven site?

**Comment by ndalliard** [comment]
Ah true story, thats one of the purposes of git, that's maybe the reason why I didn't change the modification dates in the first place. Let's discuss that in person.

**Comment by Tschanzt** [comment]
@imscaradh hm, i took a look at the maven site and apparently it isn't displayed.

**Comment by imscaradh** [comment]
Ok, according to https://maven.apache.org/guides/development/guide-documentation-style.html it will be used by Maven as meta-information. Sorry for the confusion.

**Comment by dansel91** [comment]
I don't give a shit

**Comment by dansel91** [comment]
Just kidding. Delete it

**Comment by ndalliard** [comment]
@dansel91 Done. https://github.com/imscaradh/srs/commit/c7762c781dbd1b42128065ab1dec97261e370fc4

**Pull request merged**